Release notes for 6.11

GuiMacielPereira opened the pull request:
### Description of work
This PR contains the bulk of the edited release notes

#### Summary of work
<!-- Please provide a short, high level description of the work that was done.
-->
Went over release notes and edited them so they fit the same format.

<!-- Why has this work been done? If there is no linked issue please provide appropriate context for this work.
#### Purpose of work
This can be removed if a github issue is referenced below
-->


<!-- If the original issue was raised by a user they should be named here. Do not leak email addresses
**Report to:** [user name]
-->

#### Further detail of work
<!-- Please provide a more detailed description of the work that has been undertaken.
-->

### To test:
Read over release notes to correct any typos.
Build and open release notes to check for formatting issues.

<!-- Instructions for testing.
There should be sufficient instructions for someone unfamiliar with the application to test - unless a specific
reviewer is requested.
If instructions for replicating the fault are contained in the linked issue then it is OK to refer back to these.
-->

<!-- delete this if you added release notes
*This does not require release notes* because **fill in an explanation of why**
If you add release notes please save them as a separate file using the Issue or PR number as the file name. Check the file is located in the correct directory for your note(s).
-->

<!-- Ensure the base of this PR is correct (e.g. release-next or main)
Finally, don't forget to add the appropriate labels, milestones, etc.!  -->

---

### Reviewer

Please comment on the points listed below ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)).
**Your comments will be used as part of the gatekeeper process, so please comment clearly on what you have checked during your review.** If changes are made to the PR during the review process then your final comment will be the most important for gatekeepers. In this comment you should make it clear why any earlier review is still valid, or confirm that all requested changes have been addressed.

#### Code Review

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?
- Do the release notes conform to the [release notes guide](https://developer.mantidproject.org/Standards/ReleaseNotesGuide.html)?

#### Functional Tests

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.

### Gatekeeper

If you need to request changes to a PR then please add a comment and set the review status to "Request changes". This will stop the PR from showing up in the list for other gatekeepers.

GuiMacielPereira committed:
Add outputs of running release_editor.py

Ran release_editor.py script and commited the resulting changes

GuiMacielPereira committed:
Edit diffraction.rst

GuiMacielPereira committed:
Edit direct_geometry.rst

GuiMacielPereira committed:
Edit framework.rst

GuiMacielPereira committed:
Edit indirect_geometry.rst

GuiMacielPereira committed:
Edit inelastic.rst

GuiMacielPereira committed:
Edit mantidworkbench.rst

GuiMacielPereira committed:
Edit muon.rst

GuiMacielPereira committed:
Edit reflectometry.rst

GuiMacielPereira committed:
Edit sans.rst

GuiMacielPereira committed:
Add images to diffraction.rst

GuiMacielPereira committed:
Fix warnings in building docs

GuiMacielPereira committed:
Add figures to framework.rst

GuiMacielPereira committed:
Changed place of CompareWorkspaces release note

GuiMacielPereira committed:
Add figures to inelastic.rst

GuiMacielPereira committed:
Add figure to mantidworkbench.rst

GuiMacielPereira committed:
Add figure to reflectometry.rst

GuiMacielPereira committed:
Add figure to sans.rst

GuiMacielPereira committed:
Add highlights to index.rst

- Added highlights
- Added figure

GuiMacielPereira commented:
There is a `TODO` on `index.rst` as a reminder to add a paragraph about the mantid developer package. @thomashampson I think you would be the best person to write that?

jhaigh0 commented:
This should be pointing into `release-next`

jhaigh0 reviewed (changes requested):
lots of minor things in this review. Make sure to go to the files tab and select 'Add suggestion to batch' for the one you want to accept, rather than making a commit for each.

RichardWaiteSTFC commented on docs/source/release/v6.11.0/diffraction.rst:
```diff
@@ -10,34 +10,79 @@ Powder Diffraction
+
+- Algorithm :ref:`ShowPossibleCells <algm-ShowPossibleCells>` now has a new output property, ``Cells``, that includes the cell information in a usable way.
+- New algorithm :ref:`SaveMDHistoToVTK <algm-SaveMDHistoToVTK>` that saves a :ref:`MDHistoWorkspace <MDHistoWorkspace>` as a ``VTK`` file so that it can be visualized by ``Paraview``.
+- :ref:`IntegratePeaks1DProfile <algm-IntegratePeaks1DProfile>` has improved optimisation for shoebox position - would previously be centred on nearby stronger peaks if present.
```
Sorry had a typo here - also missing release note form PR ##37646 which dramatically sped up `IntegratePeaks1DProfile`

```suggestion
- :ref:`IntegratePeaksShoeboxTOF <algm-IntegratePeaksShoeboxTOF>` has improved optimisation for shoebox position - would previously be centred on nearby stronger peaks if present.
- Execution time of :ref:`IntegratePeaks1DProfile <algm-IntegratePeaks1DProfile>` has been sped up by an order of magnitude.
```

RichardWaiteSTFC commented on docs/source/release/v6.11.0/diffraction.rst:
```diff
@@ -10,34 +10,79 @@ Powder Diffraction
 ############
-.. amalgamate:: Diffraction/Single_Crystal/Bugfixes
+- :ref:`LoadWANDSCD <algm-LoadWANDSCD>` now keeps the sgl/sgu angles set by the goniometer.
+- :ref:`ConverWANDSCDToQ <algm-ConvertWANDSCDToQ>` simplifies the underlying code by using histograms directly.
```
@zjmorgan - do you  know anything about this?

RichardWaiteSTFC commented on docs/source/release/v6.11.0/diffraction.rst:
```diff
@@ -10,34 +10,79 @@ Powder Diffraction
+  This is achieved by forcing skew > 0 (minimum skew we would expect in background).
+  This change stops peak mask of :ref:`IntegratePeaksSkew <algm-IntegratePeaksSkew>` including too many pixels due to small diffuse scattering:
+
+.. figure::  ../../images/6_11_release/single-crystal-peak-background-edited.png
+   :width: 500px
```
Sorry that's my fault - think that image was lifted form the PR - I put the 'before' images as an afterthought...

thomashampson commented on docs/source/release/v6.11.0/diffraction.rst:
```diff
@@ -10,34 +10,79 @@ Powder Diffraction
+- Algorithm :ref:`LoadEventAndCompress <algm-LoadEventAndCompress>` now takes advantage of new compression technique in :ref:`LoadEventNexus <algm-LoadEventNexus>`.
+- Algorithms :ref:`FitPeaks <algm-FitPeaks>` and :ref:`PDCalibration <algm-PDCalibration>` now do a check after the fit for the minimum signal-to-sigma ratio, where peaks with a signal below the provided threshold will be rejected.
+- :ref:`HRPD reduction scripts <isis-powder-diffraction-hrpd-ref>` now support boolean option ``fit_prompt_pulse`` to fit and subtract prompt pulse (as opposed to default method of masking of prompt pulse in TOF).
+  You can either specify this option in the configuration file or in ``hrpd.HRPD(fit_prompt_pulse=True)``.
```
```suggestion
  You can either specify this option in the configuration file or with ``hrpd.HRPD(fit_prompt_pulse=True)``.
```

thomashampson commented on docs/source/release/v6.11.0/diffraction.rst:
```diff
@@ -10,34 +10,79 @@ Powder Diffraction
 Bugfixes
 ############
-.. amalgamate:: Diffraction/Engineering/Bugfixes
+- :ref:`Fitting tab <ui engineering fitting>` of :ref:`Engineering Diffraction interface<Engineering_Diffraction-ref>` no longer crashes when a fit is re-run for a function like ``PseudoVoigt`` that contains ``FWHM`` parameter as the peak function.
```
```suggestion
- The :ref:`fitting tab <ui engineering fitting>` of the :ref:`Engineering Diffraction interface<Engineering_Diffraction-ref>` no longer crashes when a fit is re-run for a function like a ``PseudoVoigt`` that contains a ``FWHM`` parameter as the peak function.
```

thomashampson commented on docs/source/release/v6.11.0/diffraction.rst:
```diff
@@ -10,34 +10,79 @@ Powder Diffraction
+  By default the correction will be applied if class has property ``scale_integrated = True``.
+- Algorithm :ref:`AddAbsorptionWeightedPathLengths <algm-AddAbsorptionWeightedPathLengths>` now allows lean peaks to be used.
+  There is also an option to apply the correction.
+- New algorithm :ref:`FindMultipleUMatrices <algm-FindMultipleUMatrices>` to find multiple UB matrices (given lattice parameters) for sample with multiple domains or spurious peaks.
```
```suggestion
- New algorithm :ref:`FindMultipleUMatrices <algm-FindMultipleUMatrices>` to find multiple UB matrices (given lattice parameters) for samples with multiple domains or spurious peaks.
```

thomashampson commented on docs/source/release/v6.11.0/diffraction.rst:
```diff
@@ -10,34 +10,79 @@ Powder Diffraction
+
+  - This algorithm will use the lattice parameters and spacegroup provided to optimise a number (NumberOfUBs)
+    of UB matrices (B is hard-coded due to the lattice parameters provided) and return a group of peak workspaces
+    one for each UB, containing the peaks that are indexed most accurately by that UB.
```
```suggestion
    (one for each UB) containing the peaks that are indexed most accurately by that UB.
```

thomashampson commented on docs/source/release/v6.11.0/diffraction.rst:
```diff
@@ -10,34 +10,79 @@ Powder Diffraction
+  - This algorithm will use the lattice parameters and spacegroup provided to optimise a number (NumberOfUBs)
+    of UB matrices (B is hard-coded due to the lattice parameters provided) and return a group of peak workspaces
+    one for each UB, containing the peaks that are indexed most accurately by that UB.
+    This algorithm is useful for finding a single UB in the presence of spurious peaks, or finding multiple UBs when there
```
```suggestion
    The algorithm is useful for finding a single UB in the presence of spurious peaks, or finding multiple UBs when there
```

thomashampson commented on docs/source/release/v6.11.0/direct_geometry.rst:
```diff
@@ -10,35 +10,27 @@ General
 Bugfixes
 ############
-.. amalgamate:: Direct_Geometry/General/Bugfixes
+- :ref:`ALFView <ALFView-ref>` not longer crashs when moving the cursor over the Instrument View while loading data.
```
```suggestion
- :ref:`ALFView <ALFView-ref>` no longer crashs when moving the cursor over the Instrument View while loading data.
```

thomashampson commented on docs/source/release/v6.11.0/direct_geometry.rst:
```diff
@@ -10,35 +10,27 @@ General
 ############
-.. amalgamate:: Direct_Geometry/General/Bugfixes
+- :ref:`ALFView <ALFView-ref>` not longer crashs when moving the cursor over the Instrument View while loading data.
+- :ref:`PyChop` no longer uses random phase values on Merling when instrument scientist mode disabled.
```
```suggestion
- :ref:`PyChop` no longer uses random phase values on Merlin when instrument scientist mode disabled.
```

thomashampson commented on docs/source/release/v6.11.0/framework.rst:
```diff
@@ -10,66 +10,103 @@ Algorithms
+- Algorithm :ref:`algm-Load` will now give a warning if the extension specified on the filename, eg. `MUSR15189.txt`, is not found.
+  Loading files from interfaces or other algorithms should also give this warning.
+  The algorithm will still load the file by looking for other extensions, as was the case before.
+- Algorithm :ref:`GenerateGroupingPowder <algm-GenerateGroupingPowder>` now has a newer version 2 that will save the grouping file with groups starting at 1 instead of 0 to make them consisted with ``GroupingWorkspace``.
```
```suggestion
- Version 2 of algorithm :ref:`GenerateGroupingPowder <algm-GenerateGroupingPowder>` that will save the grouping file with groups starting at 1 instead of 0 to make them consisted with ``GroupingWorkspace``.
```

thomashampson commented on docs/source/release/v6.11.0/direct_geometry.rst:
```diff
@@ -10,35 +10,27 @@ General
 ############
-.. amalgamate:: Direct_Geometry/General/Bugfixes
+- :ref:`ALFView <ALFView-ref>` not longer crashs when moving the cursor over the Instrument View while loading data.
+- :ref:`PyChop` no longer uses random phase values on Merling when instrument scientist mode disabled.
```
I would think so

thomashampson commented on docs/source/release/v6.11.0/framework.rst:
```diff
@@ -10,66 +10,103 @@ Algorithms
+- Algoritm :ref:`algm-CombineDiffCal` has improved time-scaling performance and new extra validations.
+- Algorithm :ref:`CompareWorkspaces <algm-CompareWorkspaces>` now supports :ref:`Ragged Workspaces <Ragged_Workspace>`.
+- Binary operations :ref:`Plus <algm-Plus>`, :ref:`Minus <algm-Minus>`, :ref:`Divide <algm-Divide>` and :ref:`Multiply <algm-Multiply>` now support :ref:`Ragged Workspaces <Ragged_Workspace>`.
+- Algorithm :ref:`CompressEvents <algm-CompressEvents>` now has new property ``SortFirst`` that controls whether sorting first happens for compress events.
```
```suggestion
- Algorithm :ref:`CompressEvents <algm-CompressEvents>` a new ``SortFirst`` property that controls whether sorting  happens first for compress events.
```

thomashampson commented on docs/source/release/v6.11.0/framework.rst:
```diff
@@ -10,66 +10,103 @@ Algorithms
+- Algorithm :ref:`CompareWorkspaces <algm-CompareWorkspaces>` now supports :ref:`Ragged Workspaces <Ragged_Workspace>`.
+- Binary operations :ref:`Plus <algm-Plus>`, :ref:`Minus <algm-Minus>`, :ref:`Divide <algm-Divide>` and :ref:`Multiply <algm-Multiply>` now support :ref:`Ragged Workspaces <Ragged_Workspace>`.
+- Algorithm :ref:`CompressEvents <algm-CompressEvents>` now has new property ``SortFirst`` that controls whether sorting first happens for compress events.
+  If ``SortFirst=False`` then a different method, that will not sort events first, will be used to compress events which is faster when you have a large number of events per compress tolerance.
```
```suggestion
  If ``SortFirst=False`` then a different method is used to compress events that will not sort first. This is faster when you have a large number of events per compress tolerance.
```

thomashampson commented on docs/source/release/v6.11.0/framework.rst:
```diff
@@ -10,66 +10,103 @@ Algorithms
+- Binary operations :ref:`Plus <algm-Plus>`, :ref:`Minus <algm-Minus>`, :ref:`Divide <algm-Divide>` and :ref:`Multiply <algm-Multiply>` now support :ref:`Ragged Workspaces <Ragged_Workspace>`.
+- Algorithm :ref:`CompressEvents <algm-CompressEvents>` now has new property ``SortFirst`` that controls whether sorting first happens for compress events.
+  If ``SortFirst=False`` then a different method, that will not sort events first, will be used to compress events which is faster when you have a large number of events per compress tolerance.
+- Algorithm :ref:`ISISIndirectEnergyTransfer <algm-ISISIndirectEnergyTransfer>` has new property ``OutputSuffix`` that will append a suffix to the end of output workspace names.
```
```suggestion
- Algorithm :ref:`ISISIndirectEnergyTransfer <algm-ISISIndirectEnergyTransfer>` has the new property ``OutputSuffix`` that will append a suffix to the end of output workspace names.
```

thomashampson commented on docs/source/release/v6.11.0/framework.rst:
```diff
@@ -10,66 +10,103 @@ Algorithms
+- Algorithm :ref:`CompressEvents <algm-CompressEvents>` now has new property ``SortFirst`` that controls whether sorting first happens for compress events.
+  If ``SortFirst=False`` then a different method, that will not sort events first, will be used to compress events which is faster when you have a large number of events per compress tolerance.
+- Algorithm :ref:`ISISIndirectEnergyTransfer <algm-ISISIndirectEnergyTransfer>` has new property ``OutputSuffix`` that will append a suffix to the end of output workspace names.
+- Algorithms :ref:`AlignAndFocusPowderFromFiles <algm-AlignAndFocusPowderFromFiles>` and ref: :ref:`SNSPowderReduction <algm-SNSPowderReduction>` have new property ``MinSizeCompressOnLoad`` for specifying load compression.
```
```suggestion
- Algorithms :ref:`AlignAndFocusPowderFromFiles <algm-AlignAndFocusPowderFromFiles>` and ref: :ref:`SNSPowderReduction <algm-SNSPowderReduction>` have new a property called ``MinSizeCompressOnLoad`` for specifying load compression.
```

thomashampson commented on docs/source/release/v6.11.0/framework.rst:
```diff
@@ -10,66 +10,103 @@ Algorithms
+  See reference documentation here (https://docs.scipy.org/doc/scipy/reference/generated/scipy.interpolate.interp2d.html).
+- Algorithm :ref:`DiscusMultipleScatteringCorrection <algm-DiscusMultipleScatteringCorrection-v1>` now supports a radial collimator that restricts scatter points within a small region within the larger sample volume.
+  The algorithm was modified to assign zero weight to tracks where the final scatter is not in a position that allows the final track segment to pass through the collimator toward detectors.
+- Algorithm :ref:`LoadEventAsWorkspace2D <algm-LoadEventAsWorkspace2D>` has new version 2 that adds property ``FilterByTime``.
```
```suggestion
- Version 2 of the algorithm :ref:`LoadEventAsWorkspace2D <algm-LoadEventAsWorkspace2D>` that adds the property ``FilterByTime``.
```

thomashampson commented on docs/source/release/v6.11.0/framework.rst:
```diff
@@ -10,66 +10,103 @@ Algorithms
+  .. figure::  ../../images/6_11_release/ScaleInstrumentComponent.png
+     :width: 400px
+
+- Algorithm :ref:`GenerateGroupingSNSInelastic <algm-GenerateGroupingSNSInelastic>` has new input option :ref:`InstrumentDefinitionFile`.
```
```suggestion
- Algorithm :ref:`GenerateGroupingSNSInelastic <algm-GenerateGroupingSNSInelastic>` has the new input option :ref:`InstrumentDefinitionFile`.
```

thomashampson commented on docs/source/release/v6.11.0/framework.rst:
```diff
@@ -10,66 +10,103 @@ Algorithms
+     :width: 400px
+
+- Algorithm :ref:`GenerateGroupingSNSInelastic <algm-GenerateGroupingSNSInelastic>` has new input option :ref:`InstrumentDefinitionFile`.
+  Selecting the new ``InstrumentDefinitionFile`` option in instrument drop down menu will create a new field allowing users selecting older instrument definition files.
```
```suggestion
  Selecting the new ``InstrumentDefinitionFile`` option in the instrument drop down menu will create a new field allowing users to select older instrument definition files.
```

thomashampson commented on docs/source/release/v6.11.0/framework.rst:
```diff
@@ -10,66 +10,103 @@ Algorithms
+
+- Algorithm :ref:`GenerateGroupingSNSInelastic <algm-GenerateGroupingSNSInelastic>` has new input option :ref:`InstrumentDefinitionFile`.
+  Selecting the new ``InstrumentDefinitionFile`` option in instrument drop down menu will create a new field allowing users selecting older instrument definition files.
+- Algorithm :ref:`CompareWorkspaces <algm-CompareWorkspaces>` has new property ``CheckUncertainty`` to turn off comparing the y-value uncertainties.
```
```suggestion
- Algorithm :ref:`CompareWorkspaces <algm-CompareWorkspaces>` has a new ``CheckUncertainty`` property to turn off comparing the y-value uncertainties.
```

thomashampson commented on docs/source/release/v6.11.0/framework.rst:
```diff
@@ -10,66 +10,103 @@ Algorithms
 ############
-.. amalgamate:: Framework/Algorithms/Bugfixes
+- Algorithm :ref:`CompareWorkspaces <algm-CompareWorkspaces-v1>` is now fixed for relative differences of small values.
+- Algorithm :ref:`LoadEventNexus <algm-LoadEventNexus>` now has minimum histogram bin edge equal to the lowest time-of-flight event rather than one less.
```
```suggestion
- Algorithm :ref:`LoadEventNexus <algm-LoadEventNexus>` now has the minimum histogram bin edge equal to the lowest time-of-flight event rather than one less.
```

thomashampson commented on docs/source/release/v6.11.0/framework.rst:
```diff
@@ -10,66 +10,103 @@ Algorithms
 ############
-.. amalgamate:: Framework/Algorithms/Bugfixes
+- Algorithm :ref:`CompareWorkspaces <algm-CompareWorkspaces-v1>` is now fixed for relative differences of small values.
+- Algorithm :ref:`LoadEventNexus <algm-LoadEventNexus>` now has minimum histogram bin edge equal to the lowest time-of-flight event rather than one less.
+  The effect is that there is no longer negative bin edges unless there is actually a negative time-of-flight in the data.
```
```suggestion
- Algorithm :ref:`LoadEventNexus <algm-LoadEventNexus>` now has the minimum histogram bin edge equal to the lowest time-of-flight event rather than one less.
  There are no longer negative bin edges unless there is actually a negative time-of-flight in the data.
```

thomashampson commented on docs/source/release/v6.11.0/framework.rst:
```diff
@@ -10,66 +10,103 @@ Algorithms
+- Algorithm :ref:`LoadEventNexus <algm-LoadEventNexus>` now has minimum histogram bin edge equal to the lowest time-of-flight event rather than one less.
+  The effect is that there is no longer negative bin edges unless there is actually a negative time-of-flight in the data.
+- Isotope densities have been updated, see https://pypi.org/project/periodictable/1.6.1/#history for notes about the updates from NIST.
+- Algorithm `IntegratePeaks1DProfile <algm-IntegratePeaks1DProfile-v1>` no longer references `numpy.distutils`, since as of NumPy 1.23.0 it is deprecated.
```
yes, this probably is not needed for the user.

RichardWaiteSTFC replied:
Probably, they used to see a warning they no longer do I suppose?

thomashampson committed:
Release notes grammatical fixes

Co-authored-by: Jonathan Haigh <35813666+jhaigh0@users.noreply.github.com>
Co-authored-by: RichardWaiteSTFC <55979119+RichardWaiteSTFC@users.noreply.github.com>

thomashampson commented on docs/source/release/v6.11.0/inelastic.rst:
```diff
@@ -7,23 +7,54 @@ Inelastic Changes
+- :ref:`Convolution <convfit>` :ref:`QENS Fitting interface <interface-inelastic-qens-fitting>` interface no longer crashes when attempting to Fix all ``IsoDiffRot`` parameters from the ``EditLocalParameter`` dialog.
+- Fixed comparison operation in ``ISISIndirectInelastic.OSIRISIqtAndIqtFitMulti`` system test by turning off uncertainty check, using absolute comparison, and adjusting the tolerance.
+  This change does not affect general users.
+- When the ADS is cleared from workspaces that are used to run fits on an open :ref:`QENS Fitting interface <interface-inelastic-qens-fitting>`, a warning message now pops up when clicking on the `Run` button.
```
```suggestion
- When the ADS is cleared of workspaces that are used to run fits on an open :ref:`QENS Fitting interface <interface-inelastic-qens-fitting>`, a warning message now pops up when clicking on the `Run` button.
```

thomashampson commented on docs/source/release/v6.11.0/inelastic.rst:
```diff
@@ -7,23 +7,54 @@ Inelastic Changes
+  This change does not affect general users.
+- When the ADS is cleared from workspaces that are used to run fits on an open :ref:`QENS Fitting interface <interface-inelastic-qens-fitting>`, a warning message now pops up when clicking on the `Run` button.
+- Fixed a bug in the Monte Carlo error calculation on the I(Q, t) tab of the :ref:`Data Processor Interface <interface-inelastic-data-processor>` where the first bin had an error of zero.
+- :ref:`Elwin Tab <elwin>` of the :ref:`Data Processor Interface <interface-inelastic-data-processor>` now supports loading data unrestricted by suffix if the option is selected from ``Settings``.
+- :ref:`Elwin interface <elwin>` now allows to see the Spectrum number 0 on the widget plot of the selected preview workspace.
+- :ref:`Elwin interface <elwin>` now plots the correct spectrum for the selected index when changing the preview spectrum above the plot widget combo box.
```
```suggestion
- The :ref:`Elwin Tab <elwin>` of the :ref:`Data Processor Interface <interface-inelastic-data-processor>` now supports loading data unrestricted by suffix if the option is selected from ``Settings``.
- The :ref:`Elwin interface <elwin>` now allows to see the Spectrum number 0 on the widget plot of the selected preview workspace.
- The :ref:`Elwin interface <elwin>` now plots the correct spectrum for the selected index when changing the preview spectrum above the plot widget combo box.
```

thomashampson commented on docs/source/release/v6.11.0/inelastic.rst:
```diff
@@ -7,23 +7,54 @@ Inelastic Changes
+- :ref:`Elwin interface <elwin>` now allows to see the Spectrum number 0 on the widget plot of the selected preview workspace.
+- :ref:`Elwin interface <elwin>` now plots the correct spectrum for the selected index when changing the preview spectrum above the plot widget combo box.
+- :ref:`Moments<inelastic-moments>` tab of the :ref:`Data Processor <interface-inelastic-data-processor>` interface now have responsive sliders to changes in `Emin` and `EMax` properties when changed from the property browser.
+- :ref:`Inelastic Bayes fitting <interface-inelastic-bayes-fitting>` interface now correctly calculates EISF errors on the Quasi tab.
```
```suggestion
- The :ref:`Inelastic Bayes fitting <interface-inelastic-bayes-fitting>` interface now correctly calculates EISF errors on the Quasi tab.
```

thomashampson commented on docs/source/release/v6.11.0/inelastic.rst:
```diff
@@ -7,23 +7,54 @@ Inelastic Changes
+- :ref:`Moments<inelastic-moments>` tab of the :ref:`Data Processor <interface-inelastic-data-processor>` interface now have responsive sliders to changes in `Emin` and `EMax` properties when changed from the property browser.
+- :ref:`Inelastic Bayes fitting <interface-inelastic-bayes-fitting>` interface now correctly calculates EISF errors on the Quasi tab.
+- Adding new data to the  :ref:`Elwin data table <elwin>` after clearing the Analysis Data Service no longer raises a ``No data found`` warning.
+- Ploting  a preview of the selected workspace on the :ref:`Elwin tab <elwin>` no longer crashes Mantid after that workspace has been deleted from the ADS.
```
```suggestion
- Plotting a preview of the selected workspace on the :ref:`Elwin tab <elwin>` no longer crashes Mantid after that workspace has been deleted from the ADS.
```

thomashampson commented on docs/source/release/v6.11.0/inelastic.rst:
```diff
@@ -7,23 +7,54 @@ Inelastic Changes
+- :ref:`Inelastic Bayes fitting <interface-inelastic-bayes-fitting>` interface now correctly calculates EISF errors on the Quasi tab.
+- Adding new data to the  :ref:`Elwin data table <elwin>` after clearing the Analysis Data Service no longer raises a ``No data found`` warning.
+- Ploting  a preview of the selected workspace on the :ref:`Elwin tab <elwin>` no longer crashes Mantid after that workspace has been deleted from the ADS.
+- Fix a cutoff issue with ``Symmetric Energy Range`` label in :ref:`Iqt<iqt>` tab of the :ref:`Data Processor <interface-inelastic-data-processor>` interface.
```
```suggestion
- Fix a cutoff issue with ``Symmetric Energy Range`` label in the :ref:`Iqt<iqt>` tab of the :ref:`Data Processor <interface-inelastic-data-processor>` interface.
```

thomashampson commented on docs/source/release/v6.11.0/inelastic.rst:
```diff
@@ -7,23 +7,54 @@ Inelastic Changes
+- Adding new data to the  :ref:`Elwin data table <elwin>` after clearing the Analysis Data Service no longer raises a ``No data found`` warning.
+- Ploting  a preview of the selected workspace on the :ref:`Elwin tab <elwin>` no longer crashes Mantid after that workspace has been deleted from the ADS.
+- Fix a cutoff issue with ``Symmetric Energy Range`` label in :ref:`Iqt<iqt>` tab of the :ref:`Data Processor <interface-inelastic-data-processor>` interface.
+- The dialog window for adding data in the  :ref:`Elwin Tab <elwin>` of the :ref:`Data Processor Interface <interface-inelastic-data-processor>` no longer freezes when adding data.
```
```suggestion
- The dialog window for adding data in the :ref:`Elwin Tab <elwin>` of the :ref:`Data Processor Interface <interface-inelastic-data-processor>` no longer freezes when adding data.
```

thomashampson committed:
More grammatical fixes for release notes.

Co-authored-by: Jonathan Haigh <35813666+jhaigh0@users.noreply.github.com>

thomashampson reviewed (changes requested):
some more small changes and a question for @jclarkeSTFC

thomashampson committed:
Add note for mantid-developer metapackage

Although it's not directly applicable for users, it's a noteworthy change for developers.

thomashampson committed:
Final grammatical fixes in release notes

thomashampson committed:
move macOS compiler release notes into framework

thomashampson commented on docs/source/release/v6.11.0/mantidworkbench.rst:
```diff
@@ -7,36 +7,60 @@ Mantid Workbench Changes
 New Features
 ------------
-.. amalgamate:: Workbench/New_features
+- Updated compiler on macOS from clang version 15 to 16, which should result in performance improvements.
```
done

thomashampson commented:
I'm happy for this to be merged pending one change to this line, which is a bit confusing:
`- All sample times are now allowed to be before 01/01/1991.`

jclarkeSTFC commented on docs/source/release/v6.11.0/framework.rst:
```diff
@@ -10,66 +10,102 @@ Algorithms
+- Algorithm :ref:`LoadEventNexus <algm-LoadEventNexus>` now has the minimum histogram bin edge equal to the lowest time-of-flight event rather than one less.
+  There are no longer negative bin edges unless there is actually a negative time-of-flight in the data.
+- Isotope densities have been updated, see https://pypi.org/project/periodictable/1.6.1/#history for notes about the updates from NIST.
+- All sample times are now allowed to be before 01/01/1991.
```
```suggestion
- Files where all sample times are before 01/01/1991 will no longer generate an error.
```

thomashampson committed:
Clarify bug fix in release notes

Co-authored-by: James Clarke <139879523+jclarkeSTFC@users.noreply.github.com>

thomashampson commented on docs/source/release/v6.11.0/inelastic.rst:
```diff
@@ -7,23 +7,52 @@ Inelastic Changes
 New features
 ############
-.. amalgamate:: Inelastic/Algorithms/New_features
+- :ref:`Abins <algm-Abins>`/:ref:`Abins2D <algm-Abins2D>`algorithms now support "high-precision" eigenvectors from GAUSSIAN.
```
```suggestion
- :ref:`Abins <algm-Abins>`/:ref:`Abins2D <algm-Abins2D>` algorithms now support "high-precision" eigenvectors from GAUSSIAN.
```

thomashampson committed:
Fix formatting issue to avoid sphinx warning

thomashampson committed:
Fix indentation for images in release notes

thomashampson committed:
Fix order of before-after image

thomashampson reviewed (approved):
All requested changes have been implemented and docs build is fine.
